Prox/Prox/Utilities/CategoriesUtil.swift

```diff
@@ -131,8 +131,12 @@ struct CategoriesUtil {
         var categoryToFilter = [String: PlaceFilter]()
 
         // Build the initial map of categories -> filters that we set explicitly.
-        for (filter, categories) in PlaceFilter.categories {
+        for (var filter, categories) in PlaceFilter.categories {
             for category in categories {
+                // Remap Discover -> Services for now.
```

Comment:
nit: Might want to explain a bit more, in case this code lasts longer than we'll remember what this meant.